tests/system/python/e2e/test_e2e_expr_pi.py

```diff
@@ -31,6 +31,7 @@
 ASSET_NAME = "Expression"
 
 
+@pytest.mark.skip(reason="FOGL-2883: Flaky test, to be fixed")
```

Comment:
Can we skip based on Redhat platform condition?

Reply:
and more to it, can we skip just sent verification instead of full test. I'm not sure if this is not able to send data or verify sent

Reply:
fail randomly on ub16/18 as well. Nothing specific to a particular environment

Reply:
Okay